### ISSUE TITLE
Error during the installation

### ISSUE THREAD

**Issue by kongoncharuk** [EmilePerron/git-version-strategy#1]
Hi, getting the following error when installing the bundle via composer:

```
!!
!!  Fatal error: Declaration of Emileperron\GitVersionStrategyBundle\Asset\GitVersionStrategy::getVersion($path = null) must be compatible with Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface::getVersion(string $path): string in vendor/emileperron/git-version-strategy-bundle/src/Asset/GitVersionStrategy.php on line 18
!!  PHP Fatal error:  Declaration of Emileperron\GitVersionStrategyBundle\Asset\GitVersionStrategy::getVersion($path = null) must be compatible with Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface::getVersion(string $path): string in vendor/emileperron/git-version-strategy-bundle/src/Asset/GitVersionStrategy.php on line 18
!!  Symfony\Component\ErrorHandler\Error\FatalError {#7848
!!    #message: "Compile Error: Declaration of Emileperron\GitVersionStrategyBundle\Asset\GitVersionStrategy::getVersion($path = null) must be compatible with Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface::getVersion(string $path): string"
!!    #code: 0
!!    #file: "./vendor/emileperron/git-version-strategy-bundle/src/Asset/GitVersionStrategy.php"
!!    #line: 18
!!    -error: array:4 [
!!      "type" => 64
!!      "message" => "Declaration of Emileperron\GitVersionStrategyBundle\Asset\GitVersionStrategy::getVersion($path = null) must be compatible with Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface::getVersion(string $path): string"
!!      "file" => "vendor/emileperron/git-version-strategy-bundle/src/Asset/GitVersionStrategy.php"
!!      "line" => 18
!!    ]
!!  }
!!
```

Could you perhaps take a look?

Thanks!

**Comment by EmilePerron** [comment]
Hi @kongoncharuk,

The package has not been tested for Symfony 6+, and based on your report, it looks like it isn't compatible with it in its current state.

If you would like to submit a PR to make the package compatible with Symfony 6, I'll be happy to look it over and make a new release for it. 

Cheers! :)

**Comment by kongoncharuk** [comment]
Alright, will take a look 👌

**Comment by EmilePerron** [comment]
Closed by #2 :)
